net/server.go

```diff
@@ -262,20 +262,17 @@ func (s *server) PushLog(ctx context.Context, req *pb.PushLogRequest) (*pb.PushL
 		}
 
 		// Create a new DAG service with the current transaction
-		var getter format.NodeGetter = s.peer.newDAGSyncerTxn(txn)
-		if sessionMaker, ok := getter.(SessionDAGSyncer); ok {
-			getter = sessionMaker.Session(ctx)
-		}
+		fetcher := s.peer.newDAGSyncerTxn(txn)
```

Comment:
suggestion: Same as other comments, I'd prefer it if we didnt call this `fetcher`

Reply:
✅